lnrpc/routerrpc/router.proto

```diff
@@ -227,10 +227,10 @@ message SendPaymentRequest {
     string payment_request = 5;
 
     /*
-    An upper limit on the amount of time we should spend when attempting to
-    fulfill the payment. This is expressed in seconds. If we cannot make a
-    successful payment within this time frame, an error will be returned.
-    This field must be non-zero.
+    An optional upper limit on the amount of time we should spend when
+    attempting to fulfill the payment. This is expressed in seconds. If we
+    cannot make a successful payment within this time frame, an error will be
+    returned. This field must be non-zero.
```

Comment:
we should also mention the default value of 60s is used when the field is not set, and when setting the field, a value of 0 is not allowed, and we should change the below field to `optional int32 timeout_seconds = 6`.

Then when we check it, we first check if the field is set by checking if it's nil, if set, we require it to be not zero, otherwise, use the default 60s.

Reply:
Since `timeout_seconds` is of type `int32` and not a pointer, its default value will be `0` if it is not set. Therefore, whether `timeout_seconds` is not specified or explicitly set to `0`, I have implemented logic to default it to `60 seconds`.

Please let me know if there is anything else I need to address.

Reply:
Can you run `make rpc` first? Then you'll know what i meant

Reply:
Done. Thanks for the help! I will make sure to learn from this and avoid making the same mistake in the future.